### ISSUE TITLE
Bug: drop 'archived-at' header generation

### ISSUE THREAD

**Issue by sebbASF** [apache/incubator-ponymail#487]
Currently the archiver adds an 'archived-at' header to all messages.
This changes the source, and can cause the message to get a different id when reloaded.

The arrival time is still useful, but it should be stored as meta-data in the ES database rather than by changing the source.

Note that the header does not appears in mails stored by lists.apache.org since early Jan 2018, judging by the sources of dev@ponymail

**Comment by Humbedooh** [comment]
Would it make more sense to prefix it with 'X-' (thus X-Archived-At) and have Pony Mail disregard X-* when digesting the email, going forward? I think this could solve some other theoretical inconsistencies in emails depending on the path they took.

**Comment by sebbASF** [comment]
No, I think Ponymail should not change the emails that it stores.
